code/modules/Economy/Wage-Process.dm

```diff
@@ -0,0 +1,8 @@
+var/global/datum/controller/process/wagePayout/wagePayoutController = null
+/datum/controller/process/wagePayout
+	schedule_interval = 9000 // 15 minutes
```

Comment:
So something I realised with the TEG fix is that this isn't a tickrate define multiple of some kind.
Shouldn't you do that?